nncf/experimental/onnx/algorithms/quantization/min_max_quantization.py

```diff
@@ -127,18 +129,26 @@ def get_quantization_target_points(self, model: onnx.ModelProto) -> List[ONNXTar
                 if NNCFGraphNodeType.INPUT_NODE not in quantization_point.insertion_point.target_node_name:
                     # If quantization of Input
                     if quantization_point.insertion_point.input_port_id is not None:
-                        # TODO (kshpv): need to be reconsidered:
-                        #  some operators such as Mul and Add could have activation input tensor on 0 or 1 indices
-                        # TODO (kshpv): input_port_id can be usefull in terms of quantizing only one edge.
-                        #  Some of the models could required this.
-                        outputs = onnx_graph.get_node_edges(node_name)['input'][0]
+                        # If there are several inputs to quantize
+                        if isinstance(quantization_point.insertion_point.input_port_id, list):
```

Comment:
Instead of `input_port_id` type checking, can we quantize all inputs? Whether it's one input or more, doesn't matter.

Reply:
What if there is one of the inpurt_port_id connected to the constant attributes of the node? In that case we don't have to quantize it.